src/test/run-make/raw-dylib-stdcall/lib.rs

```diff
@@ -0,0 +1,29 @@
+#![feature(raw_dylib)]
+
+#[repr(C)]
+struct S {
+    x: u8,
+    y: i32,
+}
+
+#[link(name = "extern", kind = "raw-dylib")]
+extern "stdcall" {
```

Comment:
Other newly supported ABIs also need test cases.

Reply:
Added.

One question: I haven't changed the makefile, but there seems to be something wrong with my header command.  Could you have a look and tell me what I'm missing?  When I run the test with
```
x.py test -j 4 --build=i686-pc-windows-msvc src\test\run-make\raw-dylib-alt-calling-convention
```
it ignores the test, and I'm not sure why.  Deleting line 3 of the makefile allows it to run with the same command, so I've clearly gotten something wrong somewhere.

Reply:
Fixed; apparently it needs to be `# only-i686-pc-windows-msvc` -- the "pc" is crucial, for some reason.